### DISCUSSION TITLE
What are 'min_duration_off' and 'threshold' means (segmentation) ?

### DISCUSSION THREAD

**Discussion by amitli1** [pyannote/pyannote-audio#1301] [Development]
printing the pipeline parameters (`pipeline.parameters(instantiated=True)`) gives:
```
{
'segmentation':
      {
      'min_duration_off': 0.5817029604921046,
      'threshold': 0.4442333667381752
     },
'clustering':
   {
     'method': 'centroid',
     'min_cluster_size': 15,
   'threshold': 0.7153814381597874
 }
}
```

I read the article of the segmentation model ([End-to-end speaker segmentation for overlap-aware resegmentation](https://www.isca-speech.org/archive/pdfs/interspeech_2021/bredin21_interspeech.pdf)) 
and still don't understand, what is the meaning of **min_duration_off** and **threshold** ?

**Comment by LESchuchman** [comment]
threshold (between 0 and 1) controls the aggressiveness of speaker activity detection (i.e. a higher value will result in less detected speech).
min_duration_off (in seconds) controls intra-speaker pauses.  Setting it to 2, for example, would mean only pauses of 2 seconds or more by the same speaker would start a new segment.

See more here: https://github.com/pyannote/pyannote-audio/blob/develop/tutorials/adapting_pretrained_pipeline.ipynb in output 23
